cmd/snap/cmd_snap_op.go

```diff
@@ -222,7 +223,13 @@ func (x *cmdTry) Execute([]string) error {
 	opts := &client.SnapOptions{
 		DevMode: x.DevMode,
 	}
-	changeID, err := cli.Try(name, opts)
+
+	path, err := filepath.Abs(name)
```

Comment:
should the server side check if it got an absolute path instead of strangely failing on relative? maybe it's already doing that in which case ignore me :)